bigquery/indexer.py

```diff
@@ -18,135 +18,144 @@
 
 # Log to stderr.
 logging.basicConfig(level=logging.INFO,
```

Comment:
Can you create a separate PR with formatting changes to this file, and merge into master before this PR? These diffs are hard to read.

Reply:
Done